lib/transports/ws2.js

```diff
@@ -1782,13 +1792,17 @@ class WSv2 extends EventEmitter {
 
   /**
    * @param {Object} opts
-   * @param {string} opts.pair
+   * @param {string?} opts.pair
+   * @param {string?} opts.symbol
    * @param {string} opts.cbGID - callback group id
    * @param {Method} cb
    * @see https://docs.bitfinex.com/v2/reference#ws-public-trades
    */
-  onTrades ({ symbol, cbGID }, cb) {
-    this._registerListener('trades', { 0: symbol }, PublicTrade, cbGID, cb)
+  onTrades ({ symbol, pair, cbGID }, cb) {
+    const id = pair || symbol || ''
+    const model = id[0] === 'f' ? FundingTrade : PublicTrade
+
+    this._registerListener('trades', { 0: id }, PublicTrade, cbGID, cb)
```

Comment:
`PublicTrade` here should probably be `model`. 
`model` is not referenced in the current code.

Reply:
It's there don't worry.

Reply:
Nice catch thanks :)